modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+[id="images-configuration-parameters-hcp_{context}"]
+= Image controller configuration parameters for HCP
+
+The `image.config.openshift.io/cluster` resource holds cluster-wide information about how to handle images. The canonical, and only valid name is `cluster`. Its `spec` offers the following configuration parameters.
```

Comment:
>Its `spec` offers the following configuration parameters.

I would change with

> It can be configured in ROSA HCP through ROSA CLI commands or something similar

Reply:
changed, thanks